llvm/test/CodeGen/AMDGPU/GlobalISel/regbankselect-anyext.mir

```diff
@@ -13,7 +12,8 @@ body: |
     ; CHECK: liveins: $sgpr0
     ; CHECK-NEXT: {{  $}}
     ; CHECK-NEXT: [[COPY:%[0-9]+]]:sgpr(s32) = COPY $sgpr0
-    ; CHECK-NEXT: [[ANYEXT:%[0-9]+]]:sgpr(s64) = G_ANYEXT [[COPY]](s32)
+    ; CHECK-NEXT: [[DEF:%[0-9]+]]:sgpr(s32) = G_IMPLICIT_DEF
+    ; CHECK-NEXT: [[MV:%[0-9]+]]:sgpr(s64) = G_MERGE_VALUES [[COPY]](s32), [[DEF]](s32)
```

Comment:
Why are we legalizing this G_ANYEXT to G_MERGE_VALUES, but in `anyext_s1_to_s64_scc` we generate a new `G_ANYEXT`?